sound/hybrid_font.h

```diff
@@ -548,7 +548,19 @@ class HybridFont : public SaberBase {
     return current_effect_length_;
   }
 
-
+  void SB_LowBatt() override
+  {
+    //play the fonts low battery sound if it exists
+    //TODO: if lowbatt does not exist in the font, then play a global sound (instead of a beep or a talkie)
```

Comment:
This todo can be removed, "global" sounds are handled by the font search path now.